src/renderer/components/swap/Swap.utils.test.ts

```diff
@@ -260,6 +261,39 @@ describe('components/swap/utils', () => {
     })
   })
 
+  describe('getSwapLimit', () => {
+    it('100000000 * 10% slip = 90000000 => 90000999', () => {
+      const assetAmount = baseAmount(100000000)
```

Comment:
[nitpick] Rename `assetAmount` -> `amount` (since it's a `BaseAmount`, but not an `AssetAmount`)